pxb/jenkins/build-binary-pxb24

```diff
@@ -75,6 +75,12 @@ if [[ "$COMPILER" != "default" ]]; then
     export CXX=$(echo ${COMPILER} | sed -e 's/gcc/g++/; s/clang/clang++/')
 fi
 
+# CentOS 7
```

Comment:
On oraclelinux:9 there was no lsb_release, I had to do
```
RHVER=0
if [[ -f /etc/redhat-release ]]; then
    RHVER="$(rpm --eval %rhel)"
fi

if [[ ${RHVER} -eq 7 ]]; then
    JOB_CMAKE='cmake3'
fi
```
I know that we are not supporting 5.7 on oraclelinux:9, but who knows if we will not be asked to... (discussion ongoing)